File: add/codesnippet/CPP/m-system.net.sockets.soc_26_1.cpp
   static void SendTo1()
   {
      IPHostEntry^ hostEntry = Dns::Resolve( Dns::GetHostName() );
      IPEndPoint^ endPoint = gcnew IPEndPoint( hostEntry->AddressList[ 0 ],11000 );

      Socket^ s = gcnew Socket( endPoint->Address->AddressFamily,
         SocketType::Dgram,
         ProtocolType::Udp );

      array<Byte>^ msg = Encoding::ASCII->GetBytes( "This is a test" );
      Console::WriteLine( "Sending data." );
      // This call blocks. 
      s->SendTo( msg, endPoint );
      s->Close();
   }

File: add/codesnippet/CSharp/m-system.net.sockets.soc_26_1.cs
    public static void SendTo1()
    {
        IPHostEntry hostEntry = Dns.GetHostEntry(Dns.GetHostName());
        IPEndPoint endPoint = new IPEndPoint(hostEntry.AddressList[0], 11000);

        Socket s = new Socket(endPoint.Address.AddressFamily,
            SocketType.Dgram,
            ProtocolType.Udp);
      
        byte[] msg = Encoding.ASCII.GetBytes("This is a test");
        Console.WriteLine("Sending data.");
        // This call blocks. 
        s.SendTo(msg, endPoint);
        s.Close();
    }